ui/app/pages/first-time-flow/create-password/import-with-seed-phrase/import-with-seed-phrase.component.js

```diff
@@ -195,9 +196,14 @@ export default class ImportWithSeedPhrase extends PureComponent {
     }))
   }
 
+  toggleHideSeedPhrase = () => {
+    const { hideSeedPhrase } = this.state
+    this.setState({ hideSeedPhrase: !hideSeedPhrase })
```

Comment:
Because state changes get batched together, `hideSeedPhrase` might not reflect the _current_ value when this gets executed. Any changes to state that rely upon previous state should use the callback form:

```suggestion
    this.setState(({ hideSeedPhrase }) => ({ !hideSeedPhrase }))
```